### DISCUSSION TITLE
Could anyone help to fix the Cdnfundlibrary.pm?

### DISCUSSION THREAD

**Discussion by kamanwu** [finance-quote/finance-quote#364] [Q&A] [answered]
I want to fetch quotes of Canada Mutual Funds.  I think this is the best source.  But look like this module does not work any more.  Last update is 5 years ago.

Thanks.

**Comment by bpschuck** [comment, marked as answer]
> I want to fetch quotes of Canada Mutual Funds. I think this is the best source. But look like this module does not work any more. Last update is

@kamanwu Unfortunately the module Cdnfundlibrary.pm will likely be dropped from Finance::Quote in future releases. Fundata.pm is also not working. 

Sorry.

**Comment by bpschuck** [comment]
But, if someone in the community wants to contribute they can also take a look at getting quotes from www.quotemedia.com -https://www.quotemedia.com/portal/quote?qm_symbol=NBC887%3ACA

**Comment by AbstractMethod** [comment]
I have a working version of FundLibrary (https://www.fundlibrary.com).

If you can direct me a little I can generate a PR.

**Reply by bpschuck** [reply]
@AbstractMethod,
I see you forked the Finance::Quote repo some time ago, but it is woefully out of date. Something like 100+ commits behind. I was expecting to find your version of the module in your fork's master branch, or another branch (most contributors create branches for the module or whatever changes they are working on).
I'm guessing you're not well versed in Git/GitHub?
